src/main/java/io/confluent/kafkarest/entities/ConsumerRecord.java

```diff
@@ -27,19 +27,34 @@
   @NotNull
   protected V value;
 
+  @NotNull
+  protected String topic;
+
   @Min(0)
   protected int partition;
 
   @Min(0)
   protected long offset;
 
+  public ConsumerRecord(K key, V value, String topic, int partition, long offset) {
+    this.key = key;
+    this.value = value;
+    this.topic = topic;
+    this.partition = partition;
+    this.offset = offset;
+  }
+
```

Comment:
Won't the below constructor now be invalid since the `topic` should be `@NotNull` but `this.topic` is not set? I think the existing constructors would need to be replaced. I think this shouldn't matter anyway, a quick check suggests they aren't used anymore anyway.